product-updates/knowledge-center/index.html

```diff
@@ -337,6 +337,151 @@ <h2 class="usa-card__heading" id="existing">Use an existing Login.gov Account</h
 
         <h2>What's new in TDP</h2>
 
+
+        <h3 style="margin-top: 32px;">October 8th 2024 (v 3.6.4)</h3>
+
+        <h4 style="margin-top: 16px; color: #595959;">Added:</h4>
+        <ul class="usa-icon-list">
+          <li class="usa-icon-list__item" style="margin-top: 16px;">
+            <div class="usa-icon-list__icon text-ink">
+              <img src="../img/added.svg" aria-hidden="true" alt="" style="background-color: #d3ffcc; border-radius: 5px; width: 18px; height: 18px; position: relative; top: 2px; margin-right: 5px; margin-left: 5px;"/>
+            </div>
+            <div class="usa-icon-list__content">
+              <h5 class="usa-icon-list__title">
+                Fiscal Quarter Table Explainer to the TDP Data Files Page
+              </h5>
+              <p>The records within your data files are organized by <b>calendar quarter period</b> which runs from January 1 to December 31. However, these files must be reported based on the fiscal quarter period, which runs from October 1 to September 30. For example, files that cover <b>Calendar Year 2024 Quarter 4</b> are to be reported for <b>Fiscal Year 2025 Quarter 1</b>, which is due by <i>February 14, 2025.</i></p>
+              <p>We've <b>added</b> the table below to the <a href="https://tanfdata.acf.hhs.gov/data-files">data files submission page</a> to help users map the calendar period content within the file to the reporting fiscal period.</p>  
```

Comment:
sorry the confusion but i thought this link would point to the KC page https://tdp-project-updates.app.cloud.gov/knowledge-center/uploading-data.html and that this page would be updated with how the page will look? 

And something i'm also just realizing -- was this change actually made to TDP yet? I dont recall approving a dev ticket for this change, but i did approve the design ticket related to this. 

If this change isnt actually included in a qasp-approved dev ticket, then we should table this note for a future release. @victoriaatraft @reitermb @klinkoberstar @lfrohlich

Reply:
resolved on our call - content with this is moved for [future release notes](https://app.gitbook.com/o/Yd4Wv0Fi89kSKpmakWrn/s/uAlDVLhlFIbJw6huUS2x/release-notes-workflow/future-release-notes) on gitbook.